src/util/request-manager.js

```diff
@@ -283,6 +282,7 @@ export default class RequestManager {
 
   execute(opts: RequestOptions) {
     const {params} = opts;
+    const {reporter} = this;
```

Comment:
Because params.callback, that is where the reporter is used, dont have access to this .. I also can change params.callback to an arrow function